.github/workflows/test_helper.yml

```diff
@@ -165,6 +167,7 @@ jobs:
         export-env: true
       env:
         OP_SERVICE_ACCOUNT_TOKEN: ${{ secrets.OP_SERVICE_ACCOUNT_TOKEN }}
+        CODECOV_TOKEN: "op://Data Engineering/Codecov/CODECOV_TOKEN"
```

Comment:
I got this token from our org account in Codecov and added it to our 1password vault